packages/core/admin/server/policies/isTelemetryEnabled.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { createPolicy } = require('@strapi/utils').policy;
+
+/**
+ * This policy is used for routes dealing with telemetry and analytics content.
+ * It will fails when the telemetry has been disabled on the server.
+ */
+module.exports = createPolicy({
```

Comment:
I don't see why this should forbid access if the telemetry is disabled. It should rather respond gracefully IMO